challenges/objects-arrays.js

```diff
@@ -84,7 +119,9 @@ zooAnimals = [{"animal_name":"Jackal, asiatic","population":5,"scientific_name":
 The zoo wants to display both the scientific name and the animal name in front of the habitats.  Return an array with only the animal and scientific names in it.  The individual values in the array should look like this "Name: Jackal, asiatic, Scientific: Canis aureus."
 
 */
-const animalNames = [];
+const animalNames = zooAnimals.forEach(({animal_name, scientific_name}) => {
+  animalNames.push('Name: ${animal_name}, Scientific: ${scientific_name}.');
```

Comment:
That one won't work because `animalNames` is not an array so You can't push to it. Quick solution to this would be defining an array beforehand and pushing into it like this:
```
const animalNames = [];
zooAnimals.forEach(({ animal_name, scientific_name }) => {
animalNames.push(`Name: ${animal_name}, Scientific: ${scientific_name}.`);
});
```

Also if You want to take advantage of ES6 String Literals You need to use `backtick` or it won't work.